core/connectors/sinks/postgres_sink/README.md

```diff
@@ -0,0 +1,131 @@
+# PostgreSQL Sink Connector
+
+The PostgreSQL sink connector allows you to consume messages from Iggy topics and store them in PostgreSQL databases.
+
+## Features
+
+- **Automatic Table Creation**: Optionally create tables automatically
+- **Batch Processing**: Insert messages in configurable batches for performance
+- **Metadata Storage**: Store Iggy message metadata (offset, timestamp, topic, etc.)
+- **Raw Payload Storage**: Store original message payload as JSONB
+- **Flexible Data Mapping**: Store JSON message data in structured columns
+- **Connection Pooling**: Efficient database connection management
+
+## Configuration
+
+```json
+{
+  "connection_string": "postgresql://username:password@localhost:5432/database",
+  "target_table": "iggy_messages",
+  "batch_size": 100,
+  "max_connections": 10,
+  "auto_create_table": true,
+  "include_metadata": true,
+  "store_raw_payload": true,
+  "raw_payload_column": "raw_payload"
+}
+```
+
+### Configuration Options
+
+- `connection_string`: PostgreSQL connection string
+- `target_table`: Name of the table to insert messages into
+- `batch_size`: Number of messages to insert in each batch (default: 100)
+- `max_connections`: Maximum database connections (default: 10)
+- `auto_create_table`: Automatically create the target table if it doesn't exist (default: false)
+- `include_metadata`: Include Iggy metadata columns (default: true)
+- `store_raw_payload`: Store the original message payload (default: true)
+- `raw_payload_column`: Name of the column for raw payload (default: "raw_payload")
+
+## Table Schema
+
+When `auto_create_table` is enabled, the following table structure is created:
+
+```sql
+CREATE TABLE iggy_messages (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
```

Comment:
Should we use `DEFAULT gen_random_uuid()` as this would be always taken from message ID?